packages/styles/scss/components/card-in-card/_card-in-card.scss

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/themes/mixins';
+@import '../../globals/vars';
+@import '../../globals/imports';
+@import '../../globals/utils/aspect-ratio';
+@import '../image/image';
+@import '../card/index';
+
+@mixin card-in-card {
+  :host(#{$dds-prefix}-card-in-card),
+  .#{$prefix}--card-in-card {
+    width: 100%;
+
+    &__container {
```

Comment:
I think we ran into an error with the `&` concatenations and the `:host` selectors, best to replace with `.#{$prefix}--card-in-card__container` instead

Reply:
@annawen1 That's odd. In the output, I'm seeing the concatenation coming through just fine `.bx--card-in-card__container` and the styles are being reflected, however, I'm happy to change it to `.#{$prefix}--card-in-card__container` if that's the preferred way.

Reply:
Thanks @proeung for your patience; What @annawen1 meant is that in Sass it's translated to something like `:host(#{$dds-prefix}-card-in-card)__container` that doesn't match to anything.

Reply:
^Right yeah, there were errors in the integration build bundle test last week and one of the things it was complaining about was that a parent `:host(...)` selector with the `&` concatenation didn't exist

Reply:
Makes sense! I'll make sure to make this adjustment. Appreciate both of your inputs on this.